changelogs/9.0.asciidoc

```diff
@@ -0,0 +1,26 @@
+[[apm-release-notes-9.0]]
+== APM version 9.0
+* <<apm-release-notes-9.0.0>>
+
+[float]
+[[apm-release-notes-9.0.0]]
+=== APM version 9.0.0
+
+https://github.com/elastic/apm-server/compare/v\...v9.0.0[View commits]
+
+[float]
+==== Bug fixes
+- Fix overflow in validation of `apm-server.agent.config.cache.expiration` on 32-bit architectures {pull}15216[15216]
+
+[float]
+==== Breaking Changes
```

Comment:
@kruskall can you ensure that all the breaking changes are listed here please?

Reply:
sorry I auto-merged it. @kruskall can you fix it in a follow-up PR?